x-pack/plugins/canvas/public/components/workpad_page/event_handlers.js

```diff
@@ -163,16 +139,6 @@ const handleKeyDown = (commit, e, isEditable) => {
   }
 };
 
-const handleKeyPress = (commit, e, isEditable) => {
-  const { key, target } = e;
-  const upcaseKey = key && key.toUpperCase();
-  if (isEditable && !isTextInput(target) && 'GU'.indexOf(upcaseKey) !== -1) {
```

Comment:
I see `isTextInput` is disused, is it the case that the new handler already isn't active when the input target is a text input tag?

Reply:
I tested typing in an advanced filter and it appears that the handler doesn't correctly handle the event target being an input control. I'll add the check to the key handler.